packages/beacon-node/test/unit/util/peerId.test.ts

```diff
@@ -1,14 +0,0 @@
-import {expect} from "chai";
-import {createFromPrivKey} from "peer-id";
-import {fromHexString, toHexString} from "@chainsafe/ssz";
-import {createPeerId} from "../../../src/network/index.js";
-
-describe("PeerId util - for multithread sim test", () => {
-  it("Should serialize and deserialize privKey", async () => {
-    const peerId = await createPeerId();
-    const privKey = peerId.marshalPrivKey();
-    const privKeyHex = toHexString(privKey);
-    const peerIdRecov = await createFromPrivKey(fromHexString(privKeyHex));
-    expect(peerId.toB58String()).to.equal(peerIdRecov.toB58String());
```

Comment:
Why is this test deleted?

Reply:
all functionality is moved upstream to @libp2p/peer-id and @libp2p/peer-id-factory